Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20.11.1-slim AS build
+WORKDIR /app
+COPY . ./
+RUN npm install
+RUN npx nx reset
+RUN npm run build
+
+# production environment
+FROM nginxinc/nginx-unprivileged

```

Comment:
I have no experiance with the unprivileged image but it looks promising. Mayby use a concret tag like `nginxinc/nginx-unprivileged:1.27-alpine3.20`

Reply:
Ran with your suggestion.